Project/Source/Utils/ImGuiUtils.h

```diff
@@ -111,6 +132,5 @@ inline void ImGui::ResourceSlot<ResourceTexture>(const char* label, UID* target,
 		}
 	}
 	ImGui::SameLine();
-	std::string resourceIdLabel = std::string("Id: ") + std::to_string(*target);
-	ImGui::Text(resourceIdLabel.c_str());
+	ImGui::TextUnformatted("Remove Texture");
```

Comment:
I would quit the Remove texture and keep the X with the ID: blablabla in the same line. It's more compact. Otherwise, mucho texto.
The rest is ok

Reply:
I want to work on this, and other visualization enhancements in another PR, and yeah, that was exactly my idea here :)

I should actually remove these changes from the branch

Reply:
Perfect!